mreg_cli/config.py

```diff
@@ -149,12 +149,28 @@ def start_logging(
             logging.shutdown()
             self._is_logging = False
         else:
-            logging.basicConfig(
-                filename=logfile,
-                level=logging.getLevelName(level),
-                format=LOGGING_FORMAT,
-                datefmt="%Y-%m-%d %H:%M:%S",
-            )
+            try:
+                logging.basicConfig(
+                    filename=logfile,
+                    level=logging.getLevelName(level),
+                    format=LOGGING_FORMAT,
+                    datefmt="%Y-%m-%d %H:%M:%S",
+                )
+            except PermissionError:
+                failing_logfile = logfile
+                logfile = f"/tmp/mreg-cli.{os.getuid()}.log"
```

Comment:
I know we _probably_ don't have any Windows users, but could we replace this with:

```suggestion
                tmp_dir = Path(tempfile.mkdtemp())
                logfile = tmp_dir / f"mreg-cli.{os.getuid()}.log"
```